spec/src/main/doc/core/injectionandresolution.asciidoc

```diff
@@ -346,23 +344,17 @@ The behavior of all methods declared by +java.lang.Object+, except for +toString
 
 === Dependency injection
 
-From time to time the container instantiates beans and other Java EE component classes supporting injection. The resulting instance may or may not be a _contextual instance_ as defined by <<contextual_instance>>.
+From time to time the container instantiates beans and other class supporting injection. The resulting instance may or may not be a _contextual instance_ as defined by <<contextual_instance>>.
 
-The container is required to perform dependency injection whenever it creates one of the following contextual objects:
+The container is required to perform dependency injection whenever it creates the following contextual objects:
 
-* contextual instances of session beans, and
 * contextual instances of managed beans.
 
+The container is also required to perform dependency injection whenever it instantiates the following non-contextual objects:
 
-The container is also required to perform dependency injection whenever it instantiates any of the following non-contextual objects:
-
-* non-contextual instances of session beans (for example, session beans obtained by the application from JNDI or injected using +@EJB+),
-* non-contextual instances of managed beans, and
-* instances of any other Java EE component class supporting injection.
```

Comment:
From what I understand If you request such an instance from the container from an `InjectionTarget` or `Unmanaged` object, dependency injection is not performed by the container at instantiation but when you request it with the `inject()` method. So it shouldn't be there IMO.